urnai/base/savable.py

```diff
@@ -218,8 +195,8 @@ def get_pickleable_dict(self):
 
     def restore_pickleable_attributes(self, dict_to_restore):
         for attr in dict_to_restore:
-            if hasattr(self, 'pickle_black_list'):
-                if attr not in self.pickle_black_list and attr != 'pickle_black_list':
+            if hasattr(self, 'attr_block_list'):
+                if attr not in self.attr_block_list and attr != 'attr_block_list':
                     setattr(self, attr, dict_to_restore[attr])
             else:
                 setattr(self, attr, dict_to_restore[attr])
```

Comment:
Avoid `setattr`.

Reply:
Any suggestions on how? i've found this [page](https://snyk.io/pt-BR/blog/the-dangers-of-setattr-avoiding-mass-assignment/) and it shows mutiple ways, can i choose any of them?

Reply:
You can use whichever approach you prefer. If there are only a few attributes, I recommend making it as explicit as possible. If it's a lot of attributes, I think it's better to validate the dictionary schema and then copy it.